Design.md

```diff
@@ -23,6 +23,51 @@ Currently the plan in Fedora CoreOS is to deliver content via a plain
 OSTree Repo and augment our strategy with either rojig or OCI if it
 proves useful or necessary.
 
+## Release Streams
+
+- Originally discussed in [#22](https://github.com/coreos/fedora-coreos-tracker/issues/22).
+
+### Production Refs
+
+Fedora CoreOS will have several refs for use on production machines.  At any given time, each ref will be downstream of a particular Fedora branch, and will consist of a snapshot of Fedora packages plus occasionally a backported fix.
+
+- `testing`: Periodic snapshot of the current Fedora release plus `updates`.
```

Comment:
nit should we clarify `updates` as bodhi updates ?

Reply:
Done.